gtk2_ardour/automation_time_axis.cc

```diff
@@ -174,7 +174,7 @@ AutomationTimeAxisView::AutomationTimeAxisView (
 	   as the longest automation label (see ::automation_state_changed()
 	   below). be sure to include a descender.
 	*/
-	auto_dropdown.set_sizing_text(_("Mgnual"));
```

Comment:
This _typo_ is intentional. see the comment above. The string needs to include a descender, at least in English.

"Play" is tallest (due to the descender), "Manual" widest.